test/fixtures/menu.ts

```diff
@@ -15,18 +16,23 @@ function setupTest() {
   function submitInput() {
     console.log("Enter key triggered on input");
     inputObs.set("");
-  };
+  }
   // Triggers the input menu on 'input' events, if the input has text inside.
   function inputTrigger(triggerElem: Element, ctl: PopupControl): void {
     dom.onElem(triggerElem, 'input', () => {
-      triggerElem.value.length > 0 ? ctl.open() : ctl.close();
+      (triggerElem as HTMLInputElement).value.length > 0 ? ctl.open() : ctl.close();
     });
-  };
+  }
 
   return cssExample(testId('top'),
     // tabindex makes it focusable, allowing us to test focus restore issues.
     cssButton('My Menu', menu(makeMenu, {parentSelectorToMark: '.' + cssExample.className}),
       testId('btn1'), {tabindex: "-1"}),
+    cssButton('My Contextmenu', menu(makeMenu, {
+      trigger: ['contextmenu'],
+      parentSelectorToMark: '.' + cssExample.className
+    }),
+      testId('btn2'), {tabindex: "-1"}),
```

Comment:
Alignment seems slightly off here (you could move testId & tabIndex before menu if that looks cleaner)

Reply:
Fixed.